source/util/mathUtil.js

```diff
@@ -9,7 +9,24 @@ function anyDieSucceeds(successChance, extraDice) {
 	return 1 - ((1 - successChance) ** (1 + extraDice));
 };
 
+/**
+ * @param {Set} firstSet
+ * @param {Set} secondSet
+ */
+function areSetContentsCongruent(firstSet, secondSet) {
+	if (firstSet.size !== secondSet.size) {
+		return false;
+	}
+
+	for (const element of firstSet) {
```

Comment:
I originally wanted to use isSubsetOf(), but was scared off thinking it might not be in the node version I was running because VS Code wasn't auto-completing it. (It does work in the node REPL though)